src/ripple/rpc/handlers/AccountInfo.cpp

```diff
@@ -77,6 +77,28 @@ doAccountInfo(RPC::JsonContext& context)
     if (jvAccepted)
         return jvAccepted;
 
+    static constexpr std::
+        array<std::pair<std::string_view, LedgerSpecificFlags>, 8>
+            lsFlags{
+                {{"defaultRipple", lsfDefaultRipple},
+                 {"depositAuth", lsfDepositAuth},
+                 {"disableMasterKey", lsfDisableMaster},
+                 {"globalFreeze", lsfGlobalFreeze},
+                 {"noFreeze", lsfNoFreeze},
+                 {"passwordSpent", lsfPasswordSpent},
+                 {"requireAuthorization", lsfRequireAuth},
+                 {"requireDestinationTag", lsfRequireDestTag}}};
+
+    static constexpr std::
+        array<std::pair<std::string_view, LedgerSpecificFlags>, 5>
+            disallowIncomingFlags{
+                {{"disallowIncomingXRP", lsfDisallowXRP},
```

Comment:
disallowIncomingXRP is a current flag and should not be gated by the DisallowIncoming amendment.